pkg/sql/sqlstats/sslocal/sql_stats_test.go

```diff
@@ -608,7 +608,8 @@ func TestAssociatingStmtStatsWithTxnFingerprint(t *testing.T) {
 
 			transactionFingerprintID := appstatspb.TransactionFingerprintID(txnFingerprintIDHash.Sum())
 			statsCollector.EndTransaction(ctx, transactionFingerprintID)
-			err := statsCollector.RecordTransaction(ctx, transactionFingerprintID, sqlstats.RecordedTxnStats{
+			err := statsCollector.RecordTransaction(ctx, sqlstats.RecordedTxnStats{
```

Comment:
This should be in the previous commit right?

Reply:
Ooh yeah, this should be in commit 2. Done.